index.html

```diff
@@ -362,7 +362,7 @@ <h3 class="about__connect fz-16 fw-500 lh-24 clr-p-400">
                         </li>
                         <li class="about__icons__li">
                             <a href="#" class="about__icons__link link"><img
-                                    src="images/hand.svg" alt="hand icon" /></a>
+                                    src="images/hand.svg" alt="Angilist icon" /></a>
```

Comment:
- [ ] Please remove the text "icon", even don't use the words like `button`, `icon`, `image of` in your **alternative-text** because by default when  a person who use screen-readers, screen reader automatically provide the context. Kindly use "Angelist Profile" because it will assist the person to know whether he/she is interested to look into your `profile` or not.